### PR TITLE
AEMM & Windows integrations

### PR DESCRIPTION
**Pull request by brycahta**
**Issue #, if available:** N/A

**Description of changes:**
* Using AEMM 1.2.0
* Changing Windows 10 version to 1809
* Updated Makefile and Travis for Windows builds+deployments
* Corrected `Dockerfile.windows` for webhook-test-proxy
* Updated `push-docker-images` and `upload-resources-to-github` scripts with the latest from AEMM
  * adding **binaries-only** option to `upload-to-github` is NTH-specific

**Testing:**
* ran `eks-cluster-test` for Linux and Windows on muh machine

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/upload-resources-to-github [hunk 1]
```diff
@@ -1,32 +1,103 @@
...
+ASSET_IDS_UPLOADED=()
 
+trap 'handle_errors_and_cleanup $?' EXIT
 
-## Upload k8s yaml
-resourceFiles=($TAR_RESOURCES_FILE $AGG_RESOURCES_YAML)
-for resourceFile in "${resourceFiles[@]}"; do
-    curl \
+handle_errors_and_cleanup() {
+    if [ $1 -eq "0" ]; then
```
Comment (cjerad) [inline scripts/upload-resources-to-github:47]: The `-eq` operator is for comparing numbers, but `"0"` is a string.

```suggestion
    if [ $1 -eq 0 ]; then
```

File: scripts/upload-resources-to-github [hunk 2]
```diff
@@ -1,32 +1,103 @@
...
+        exit 0
+    fi
+
+    if [[ ${#ASSET_IDS_UPLOADED[@]} -ne 0 ]]; then
+        echo -e "\nCleaning up assets uploaded in the current execution of the script"
+        for asset_id in $ASSET_IDS_UPLOADED; do
+            echo "Deleting asset $asset_id"
+            curl -X DELETE \
+            -H "Authorization: token $GITHUB_TOKEN" \
+            "https://api.github.com/repos/aws/aws-node-termination-handler/releases/assets/$asset_id"
```
Comment (cjerad) [inline scripts/upload-resources-to-github:57]: minor: indent continued lines

```suggestion
            curl -X DELETE \
                -H "Authorization: token $GITHUB_TOKEN" \
                "https://api.github.com/repos/aws/aws-node-termination-handler/releases/assets/$asset_id"
```

File: scripts/push-docker-images [hunk 3]
```diff
@@ -53,6 +53,16 @@ if [[ ${#PLATFORMS[@]} -gt 1 && $MANIFEST != "true" ]]; then
     exit 1
 fi
 
+# Existing manifests cannot be updated only overwritten; therefore,
+# if manifest exists already, fetch existing platforms so "updated" manifest includes images
+# that were there previously
+if [[ $MANIFEST == "true" ]]; then
+  manifest_exists=$(docker manifest inspect $IMAGE_REPO:$VERSION > /dev/null ; echo $?)
+  if [[ manifest_exists -eq 0 ]]; then
```
Comment (cjerad) [inline scripts/push-docker-images:61]: Missing variable expansion.

```suggestion
  if [[ $manifest_exists -eq 0 ]]; then
```

File: .travis.yml [hunk 4]
```diff
@@ -37,10 +37,20 @@ jobs:
       name: Go Report Card Test
     - stage: Test
       script: make build-binaries
-      name: Build Binaries
+      name: Build Binaries Linux
+    - stage: Test
+      name: Build Binaries Windows
+      os: windows
+      install: choco install make && choco install zip
```
[the remark this answers is not in the document]
Comment (brycahta) [reply .travis.yml:44]: I tested `choco install make && choco install zip` but not `choco install make zip` .. if they are equivalent then I can update

**Pull request merged**